governance/polls/Ratification Poll - Adding a SPF - Makershire Hathaway (MIP55c3-SP4) - June 13, 2022.md

```diff
@@ -0,0 +1,60 @@
+---
+title: Ratification Poll for Adding a SPF - Makershire Hathaway (MIP55c3-SP4) - June 13, 2022
+summary: The Makershire Hathaway SPF will be a 10M pool managed by the SF CU under the supervision of Risk CU and prospective LOVE CU to experiment with various yield generation strategies and support DAI liquidity/utility.
+discussion_link: https://forum.makerdao.com/t/mip55c3-sp4-adding-a-special-purpose-fund-makershire-hathaway/14643
+vote_type: Plurality Voting
+categories:
+   - MIPs
+   - High Impact
+options:
+   0: Abstain
+   1: Yes
+   2: No
+start_date: 2022-06-13T16:00:00
+end_date: 2022-06-27T16:00:00
+---
+# Ratification Poll for Adding a SPF - Makershire Hathaway (MIP55c3-SP4)
+
+The Governance Facilitators have placed a ratification poll into the [voting system](https://vote.makerdao.com/polling) as part of the responsibilities defined in [MIP51](https://mips.makerdao.com/mips/details/MIP51). This Governance [Poll](https://community-development.makerdao.com/en/learn/governance/on-chain-gov) will be active for fourteen days beginning on Monday, June 13 at 16:00 UTC.
+
+**This is a binary vote.**
+- **You may vote for a single option.**
+- **You should vote for the option which you prefer.**
+- **If you would accept either option, you should vote 'Abstain'.**
+
+## Review
+
+The community may vote in this poll to express support or opposition to MIP55c3-SP4 being accepted and implemented in the Maker Protocol.
+
+A brief summary of this proposal has been provided by the MIP Author and is shown below:
+
+*The Maker Protocol holds over $5.4B in stablecoins earning zero yield. There are multiple efforts underway (short-term bonds 1, IVs, RWA, Multi-chain) to invest the majority of stablecoin exposure into growth opportunities. Our aim is to begin experimenting with generating yield on balance sheet assets and support DAI liquidity/utility, creating a new revenue stream for the protocol. As DeFi is evolving quickly, we should continue researching, investigating, and experimenting with DeFi-native yield-generation strategies.*
+
+Please review the links below to inform your position on this proposal before voting.
+* [Canonical Proposal Version](https://github.com/makerdao/mips/commit/24779f52630df945595594759e4cb4b757f4c177)
```

Comment:
Use this URL: https://github.com/makerdao/mips/blob/24779f52630df945595594759e4cb4b757f4c177/MIP55/MIP55c3-Subproposals/MIP55c3-SP4.md